src/library/library.cpp

```diff
@@ -551,3 +565,13 @@ TrackCollection& Library::trackCollection() {
     DEBUG_ASSERT(m_pTrackCollectionManager->internalCollection());
     return *m_pTrackCollectionManager->internalCollection();
 }
+
+#ifdef __ENGINEPRIME__
+std::unique_ptr<mixxx::LibraryExporter> Library::makeLibraryExporter(
+        QWidget* parent) {
+    // New object is expected to be owned (and lifecycle-managed)
```

Comment:
...then a parented_ptr should be returned instead of a unqiue_ptr. The comment doesn't fit the code.